### PR TITLE
Modify the test app to test the new Distribute library

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [ ] ~Has `CHANGELOG.md` been updated?~
* [ ] ~Are tests passing locally?~
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Modify the test app so the new Distribute play store library can be tested at release time

## Related PRs or issues

[AB#81918](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/81918)

### CODE REVIEW

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Contoso.Forms.Demo.csproj [hunk 1]
```diff
@@ -8,10 +8,23 @@
     <AssemblyVersion>0.0.0.0</AssemblyVersion>
     <FileVersion>3.3.0.0</FileVersion>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' "/>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Contoso.Forms.Demo.csproj:12]: Why do you need these empty tags?

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj [hunk 2]
```diff
@@ -49,6 +49,22 @@
...
+    <WarningLevel>4</WarningLevel>
+    <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
+    <AndroidPackageFormat>apk</AndroidPackageFormat>
+    <AndroidUseAapt2>true</AndroidUseAapt2>
+    <AndroidDexTool>d8</AndroidDexTool>
+    <AndroidLinkTool>proguard</AndroidLinkTool>
+    <JavaMaximumHeapSize></JavaMaximumHeapSize>
+    <AndroidManifest></AndroidManifest>
+    <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
+    <AndroidSupportedAbis>armeabi-v7a</AndroidSupportedAbis>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj:66]: This line doesn't look valid

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/Contoso.Forms.Puppet.csproj [hunk 3]
```diff
@@ -16,6 +16,16 @@
...
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType></DebugType>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'GooglePlay|AnyCPU' ">
+    <IntermediateOutputPath>obj\GooglePlay</IntermediateOutputPath>
+    <DebugType></DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\GooglePlay</OutputPath>
+    <DefineConstants>TRACE;RELEASE;NETSTANDARD;NETSTANDARD2_0</DefineConstants>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/Contoso.Forms.Puppet.csproj:27]: Manually defining it is not correct

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/Contoso.Forms.Puppet.csproj [hunk 4]
```diff
@@ -27,10 +37,10 @@
     </ProjectReference>
     <ProjectReference Include="..\..\..\SDK\AppCenterCrashes\Microsoft.AppCenter.Crashes\Microsoft.AppCenter.Crashes.csproj">
     </ProjectReference>
-    <ProjectReference Include="..\..\..\SDK\AppCenterDistribute\Microsoft.AppCenter.Distribute\Microsoft.AppCenter.Distribute.csproj">
-    </ProjectReference>
     <ProjectReference Include="..\..\..\SDK\AppCenterPush\Microsoft.AppCenter.Push\Microsoft.AppCenter.Push.csproj">
     </ProjectReference>
+    <ProjectReference Include="..\..\..\SDK\AppCenterDistribute\Microsoft.AppCenter.Distribute\Microsoft.AppCenter.Distribute.csproj">
+    </ProjectReference>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/Contoso.Forms.Puppet.csproj:43]: What's the change here?

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/Contoso.Forms.Puppet.csproj (hunk 3 above)
Comment (AnastasiaKubova) [reply Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/Contoso.Forms.Puppet.csproj:27]: It was added via VS and the same configuration is used for release mode

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Contoso.Forms.Demo.csproj [hunk 5]
```diff
@@ -8,10 +8,21 @@
     <AssemblyVersion>0.0.0.0</AssemblyVersion>
     <FileVersion>3.3.0.0</FileVersion>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'GooglePlay|AnyCPU' ">
+    <IntermediateOutputPath>obj\GooglePlay</IntermediateOutputPath>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\GooglePlay</OutputPath>
+    <DefineConstants>TRACE;RELEASE;NETSTANDARD;NETSTANDARD2_0</DefineConstants>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Contoso.Forms.Demo.csproj:15]: Remove defining default constants

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Contoso.Forms.Demo.csproj [hunk 6]
```diff
@@ -8,10 +8,21 @@
...
+  <ItemGroup Condition=" '$(Configuration)'=='GooglePlay' ">
+    <PackageReference Include="Microsoft.AppCenter.DistributePlay" Version="3.3.0" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(Configuration)'!='GooglePlay' ">
+    <PackageReference Include="Microsoft.AppCenter.Distribute" Version="3.3.0" />
+  </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.AppCenter.Analytics" Version="3.3.0" />
     <PackageReference Include="Microsoft.AppCenter.Crashes" Version="3.3.0" />
-    <PackageReference Include="Microsoft.AppCenter.Distribute" Version="3.3.0" />
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Contoso.Forms.Demo.csproj:14]: You should not touch this project at all for this changes

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Contoso.Forms.Demo.csproj [hunk 7]
```diff
@@ -8,6 +8,11 @@
     <AssemblyVersion>0.0.0.0</AssemblyVersion>
     <FileVersion>3.3.0.0</FileVersion>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'GooglePlay|AnyCPU' ">
+    <IntermediateOutputPath>obj\GooglePlay</IntermediateOutputPath>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\GooglePlay</OutputPath>
+  </PropertyGroup>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Contoso.Forms.Demo.csproj:15]: Please revert changes in shared project

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/Contoso.Forms.Demo.Droid.csproj [hunk 8]
```diff
@@ -49,16 +49,37 @@
...
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
     <Reference Include="Mono.Android" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(Configuration)'=='GooglePlay' ">
+    <PackageReference Include="Microsoft.AppCenter.DistributePlay">
+      <Version>3.3.1-r0004-c276718</Version>
+    </PackageReference>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/Contoso.Forms.Demo.Droid.csproj:75]:
```suggestion
    <PackageReference Include="Microsoft.AppCenter.DistributePlay" Version="3.3.1-r0004-c276718" />
```

**Pull request merged**